airbyte-integrations/connectors/source-genesys/source_genesys/source.py

```diff
@@ -7,11 +7,42 @@
 from typing import Any, Dict, Iterable, List, Mapping, MutableMapping, Optional, Tuple
 
 import requests
+from datetime import datetime
+from airbyte_cdk.logger import init_logger
 from airbyte_cdk.sources import AbstractSource
 from airbyte_cdk.sources.streams import Stream
 from airbyte_cdk.sources.streams.http import HttpStream
 from source_genesys.authenicator import GenesysOAuthAuthenticator
 
+logger = init_logger()
+
+metrics = ['nBlindTransferred', 'nCobrowseSessions', 'nConnected', 'nConsult', 'nConsultTransferred', 'nError',
```

Comment:
move this to close where it's used same for query.

Reply:
All users have these fields? Why not add a new parameter in the connector spec.json where users can specify this list?

Reply:
I'm in the process of passing these with either: an optional config OR a default value.